stories/example-screen-staff-list.stories.js

```diff
@@ -289,322 +289,105 @@ const tabsDoubleRowHTML = `
       </a>
     </li>
   </ul>
-  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
-    <li class="nav-item">
-      <a class="nav-link active fs-5" href="#">
+
+  <ul class="nav nav-tabs bg-white ps-7 pe-7">
+    <li class="nav-item py-2 px-3">
+      <a class="fs-6 text-primary" href="#" style="font-weight: 400;">
         <span>All Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" aria-current="page" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
```

Comment:
We need to remove all inline styles and replace with named classes. 

Please fix all `font-weight: 400;` declarations with `fw-normal` class.